rubygem-ovirt_provision_plugin/rubygem-ovirt_provision_plugin.spec

```diff
@@ -0,0 +1,111 @@
+# This package contains macros that provide functionality relating to
+# Software Collections. These macros are not used in default
+# Fedora builds, and should not be blindly copied or enabled.
+# Specifically, the "scl" macro must not be defined in official Fedora
+# builds. For more information, see:
+# http://docs.fedoraproject.org/en-US/Fedora_Contributor_Documentation
+# /1/html/Software_Collections_Guide/index.html
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name ovirt_provision_plugin
+
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
+%global foreman_pluginconf_dir %{foreman_dir}/config/settings.plugins.d
+
+Summary:    Foreman plugin to provision new hosts and integrate to oVirt Engine
+Name:       %{?scl_prefix}rubygem-%{gem_name}
+Version:    1.0.2
+Release:    1%{?foremandist}%{?dist}
+Group:      Applications/System
+License:    GPLv3
+URL:        http://github.com/theforeman/ovirt_provision_plugin
+Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires:   foreman >= 1.11.0
+Requires:   %{?scl_prefix}rubygem(deface)
```

Comment:
Not sure if this should be an open dependency, I recall there were breaking changes pre 1.0 - https://github.com/spree/deface#upgrading-from-09-to-10